apps/src/lib/kits/maker/Piezo.js

```diff
@@ -39,6 +39,13 @@ Piezo.prototype.play = function (notes, tempo = 120) {
   });
 };
 
+Piezo.prototype.playNotes = function (notes, tempo = 120) {
+  Piezo.play(notes, tempo);
+};
+
+Piezo.prototype.playSong = function (notes, tempo = 120) {
+  Piezo.play(notes, tempo);
+};
```

Comment:
Since these are (in behavior) exactly the same as `Piezo.play` you could define them like this:
```js
Piezo.prototype.playNotes = Piezo.prototype.play;
Piezo.prototype.playSong = Piezo.prototype.play;
```